src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -159,90 +157,85 @@
-    (reject! "The type set in the authorization rule specified "
-             "as " (pprint-rule rule) " should be a "
-             "string that is either 'path' or 'regex'."))
+    (reject! (trs "The type set in the authorization rule specified as {0} should be a string that is either ''path'' or ''regex''."
+                  (pprint-rule rule))))
   (let [type (-> rule :match-request :type name str/lower-case)]
     (when-not (or (= type "path") (= type "regex"))
-      (reject! "The type set in the authorization rule specified "
-               "as " (pprint-rule rule) " is invalid. "
-               "It should be set to either 'path' or 'regex'.")))
+      (reject! (trs "The type set in the authorization rule specified as {0} is invalid. "
+                    (pprint-rule rule))
+               (trs "It should be set to either ''path'' or ''regex''."))))
   (when-not (string? (:path (:match-request rule)))
-    (reject! "The path set in the authorization rule specified as "
-             (pprint-rule rule) " is invalid. "
-             "It should be a string."))
+    (reject! (trs "The path set in the authorization rule specified as {0} is invalid. "
+                  (pprint-rule rule))
+             (trs "It should be a string.")))
   (when-not (valid-method? rule)
-    (reject! "The method specified in the authorization rule specified as "
-             (pprint-rule rule) " is invalid. "
-             "It should be either a string or list of strings that is "
-             "equal to one of the following methods: '"
-             (str/join "', '" (sort valid-methods)) "'"))
+    (reject! (trs "The method specified in the authorization rule specified as {0} is invalid. "
+                  (pprint-rule rule))
+             (trs "It should be either a string or list of strings that is equal to one of the following methods: ")
+             (format "'%s'" (str/join "', '" (sort valid-methods)))))
   (when (= "regex" (-> rule :match-request :type name str/lower-case))
     (validate-regex-backreferences! rule))
   (doseq [[type aces] (select-keys rule [:allow :deny])]
     (if (vector? aces)
       (when-not (every? #(or (string? %) (map? %)) aces)
-        (reject! "The " (name type) " list in the rule specified as "
-                 (pprint-rule rule)
-                 " contains one or more aces that are not maps or strings."))
+        (reject! (trs "The {0} list in the rule specified as {1} contains one or more ACEs that are not maps or strings."
+                      (name type) (pprint-rule rule))))
       (when-not (or (map? aces) (string? aces))
-        (reject! "The ACE '" aces "' in the '" (name type) "' field of "
-                 "the rule specified as " (pprint-rule rule) " is invalid. "
-                 "It should be a string or a map with keys :extensions or :certname."))))
+        (reject! (trs "The ACE ''{0}'' in the ''{1}'' field of the rule specified as {2} is invalid. "
```

Comment:
D'oh.  Somehow my eyes blocked out the parens around those and I wasn't looking that closely.  Nothing to see here - never mind.  🚶